application.py

```diff
@@ -376,24 +376,40 @@ def rate_yes():
      return ({'response':'Good'})
      # this method will insert into the user_rates_movie table 
 
- # ENDPOINT - User rating no
-# @application.route('/rate-no', methods = ['PUT'])
-# def rate_yes():
-#     movieID = request.json['movieID']
-#     userID = request.json['userID']
-    
-#     user_rates_movie_schema = User_Rates_Movie(movieID,userID,False)
+# ENDPOINT - User rating NO
+@application.route('/rate-no', methods = ['PUT'])
+def rate_no():
+     userID = request.json['userID']
+     movieID = request.json['movieID']
+     
+     newRecord = Userratesmovie(movieID,userID,False)
 
-#     db.session.add(user_rates_movie_schema)
-#     db.session.commit()
-#     # call method to display movie, will display a new unseen movie
+     db.session.add(newRecord)
+     db.session.commit()
+     # call method to display movie, will display a new unseen movie
 
-#     return ({'response':'Good'})
-#     # this method will insert into the user_rates_movie table 
+     return ({'response':'Good'})
+     # this method will insert into the user_rates_movie table 
 
+# ENDPOINT - User rates movie
+# This is an endpoint to consolidate the Yes and No separate endpoints, it will handle both
```

Comment:
This is good work using consolidating the yes and no endpoints with a single method.